solutions/swb-app/src/accountRoutes.ts

```diff
@@ -16,8 +25,38 @@ export function setUpAccountRoutes(router: Router, account: HostingAccountServic
     '/aws-accounts',
     wrapAsync(async (req: Request, res: Response) => {
       processValidatorResult(validate(req.body, CreateAccountSchema));
-      const response = await account.create(req.body);
-      res.send(response);
+      const { name, awsAccountId, envMgmtRoleArn, hostingAccountHandlerRoleArn, externalId } = req.body;
+      const createAccountMetadata: CreateAccountMetadata = {
+        name: escape(name),
+        awsAccountId: escape(awsAccountId),
+        envMgmtRoleArn: escape(envMgmtRoleArn),
+        hostingAccountHandlerRoleArn: escape(hostingAccountHandlerRoleArn),
+        externalId: escape(externalId)
+      };
+
+      const createdAccount = await account.create(createAccountMetadata);
+      res.send(createdAccount);
+    })
+  );
+
+  router.patch(
+    '/aws-accounts/:id',
+    wrapAsync(async (req: Request, res: Response) => {
+      processValidatorResult(validate(req.body, UpdateAccountSchema));
+      const { name, awsAccountId, envMgmtRoleArn, hostingAccountHandlerRoleArn, externalId } = req.body;
```

Comment:
@bhudnell I've added what feels like extremely gratuitous escaping logic when constructing the actual `UpdateAccountMetadata` and `CreateAccountMetadata` objects from the request body in their respective methods. For some reason we're still getting pinged for the CWE. I'm pretty sure that escaping all present data mitigates the issue, right? Or is this truly a valid CWE?

Additionally, do you know any ways to better manage sanitization of an arbitrary response object? Using `escape` was the quickest thing I could think of, but it feels like it won't scale.